cloud-formation/membership-app.cf.json

```diff
@@ -102,35 +102,28 @@
                     "Fn::Join":["\n", [
                         "#!/bin/bash -ev",
 
-                        "CONF_DIR=/membership/frontend-1.0-SNAPSHOT/conf",
+                        "CONF_DIR=/etc/frontend",
 
-                        "/opt/features/ssh-keys/initialise-keys-and-cron-job.sh -l -b github-team-keys -t Membership-and-Subscriptions || true",
-
-                        "adduser --system --home /membership --disabled-password membership",
-
-                        {"Fn::Join":["", ["aws --region ", { "Ref": "AWS::Region" }, " s3 cp s3://membership-dist/", { "Ref" : "Stack" }, "/", { "Ref" : "Stage" }, "/frontend/frontend-1.0-SNAPSHOT.tgz /tmp"]]},
-
-                        "tar xvz -C /membership -f /tmp/frontend-1.0-SNAPSHOT.tgz",
+                        {"Fn::Join":["", ["aws --region ", { "Ref": "AWS::Region" }, " s3 cp s3://membership-dist/", { "Ref" : "Stack" }, "/", { "Ref" : "Stage" }, "/frontend/frontend_1.0-SNAPSHOT_all.deb /tmp"]]},
 
                         "mkdir /etc/gu",
 
                         {"Fn::Join":["", ["aws --region ", { "Ref": "AWS::Region" }, " s3 cp s3://membership-private/", { "Ref" : "Stage" }, "/membership.private.conf /etc/gu"]]},
-                        "chown membership /etc/gu/membership.private.conf",
-                        "chmod 0600 /etc/gu/membership.private.conf",
 
                         "wget https://s3.amazonaws.com/aws-cloudwatch/downloads/latest/awslogs-agent-setup.py",
+
+                        "dpkg -i /tmp/frontend_1.0-SNAPSHOT_all.deb",
+                        "chown frontend /etc/gu/membership.private.conf",
+                        "chmod 0600 /etc/gu/membership.private.conf",
+
                         {"Fn::Join": ["", ["sed -i",
                             " -e \"s/__DATE/$(date +%F)/\"",
-                            " -e \"s/__BUILD/$(cat $CONF_DIR/build.txt)/\"",
                             " -e 's/__STAGE/", { "Ref": "Stage" }, "/'",
                             " $CONF_DIR/logger.conf"]]},
                         {"Fn::Join": [" ", ["python awslogs-agent-setup.py -nr", { "Ref": "AWS::Region" }, "-c $CONF_DIR/logger.conf" ]]},
```

Comment:
Hmm, looks they _are_ still using this Python agent on Contributions:

https://github.com/guardian/contributions-frontend/blob/9830dbf1/cloud-formation/cfn.yml#L151-L156